<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OddBall</class>
 <widget class="QWidget" name="OddBall">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OddBall</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>881</width>
     <height>581</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>100</horstretch>
     <verstretch>100</verstretch>
    </sizepolicy>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabcourt1">
    <attribute name="title">
     <string>Court 1</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>371</width>
       <height>211</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>2</number>
     </property>
     <property name="columnCount">
      <number>3</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>Frame</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Points</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>W-Team</string>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>1</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>2</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </item>
    </widget>
    <widget class="QGroupBox" name="groupBox_camsource">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>471</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Camera Feed Source</string>
     </property>
     <widget class="QRadioButton" name="radioButton_cam1">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>cam1</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_cam2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>cam2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_cam3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>cam3</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_cam7">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>cam7</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_cam4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>cam4</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_cam5">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>cam5</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_cam6">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>cam6</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_cam8">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>60</y>
        <width>100</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>cam8</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_annotations">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>471</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Annotations</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QCheckBox" name="checkBox_venue">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>86</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Venue</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_team">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>86</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Team</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_player">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>30</y>
        <width>86</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Player</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_balltrails">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>86</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ball Trails</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_score">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>86</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Score</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_replay">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>461</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Replay</string>
     </property>
     <widget class="QPushButton" name="pushButton_replay">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>112</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Replay</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_time">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>30</y>
        <width>42</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>5</number>
      </property>
      <property name="maximum">
       <number>15</number>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_throw">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Throw</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_frame">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Frame</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_time">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Time</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_camera">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>451</width>
       <height>291</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_record">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>380</y>
       <width>201</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>record active cams</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabconfig">
    <attribute name="title">
     <string>config</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_cameras">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>701</width>
       <height>271</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>600</width>
       <height>0</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>6</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>7</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>8</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Active</string>
      </property>
      <property name="textAlignment">
       <set>AlignLeading|AlignVCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Source Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Source</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Source Name</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>PubSubImageZMQCamera</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>rpi4b2gb</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>BirdsEye A</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>PubSubImageZMQCamera</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>rpi4b4gb</string>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string>BirdsEye B</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>PubSubImageZMQCamera</string>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string>rpi4b8gb</string>
      </property>
     </item>
     <item row="2" column="3">
      <property name="text">
       <string>BirdsEye C</string>
      </property>
     </item>
     <item row="3" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="3" column="1">
      <property name="text">
       <string>PubSubImageZMQCamera</string>
      </property>
     </item>
     <item row="3" column="2">
      <property name="text">
       <string>null</string>
      </property>
     </item>
     <item row="3" column="3">
      <property name="text">
       <string>PlayerSide A</string>
      </property>
     </item>
     <item row="4" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="4" column="1">
      <property name="text">
       <string>PubSubImageZMQCamera</string>
      </property>
     </item>
     <item row="4" column="2">
      <property name="text">
       <string>null</string>
      </property>
     </item>
     <item row="4" column="3">
      <property name="text">
       <string>PlayerSide C</string>
      </property>
     </item>
     <item row="5" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="5" column="1">
      <property name="text">
       <string>RTSPCamera</string>
      </property>
     </item>
     <item row="5" column="2">
      <property name="text">
       <string>rtsp://192.168.1.166</string>
      </property>
     </item>
     <item row="5" column="3">
      <property name="text">
       <string>Mobile</string>
      </property>
     </item>
     <item row="6" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsDropEnabled|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item row="6" column="1">
      <property name="text">
       <string>USBCamera</string>
      </property>
     </item>
     <item row="6" column="2">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="6" column="3">
      <property name="text">
       <string>Comment0</string>
      </property>
     </item>
     <item row="7" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="checkState">
       <enum>Checked</enum>
      </property>
     </item>
     <item row="7" column="1">
      <property name="text">
       <string>USBCamera</string>
      </property>
     </item>
     <item row="7" column="2">
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item row="7" column="3">
      <property name="text">
       <string>Comment1</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
